packages/ui/src/modal/list/ListCheckout.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC } from 'react'
+import {
+  ChainIcon,
+  Flex,
+  FormatCryptoCurrency,
+  Img,
+  Text,
+} from '../../primitives'
+import { useCollections, useTimeSince, useTokens } from '../../hooks'
+import { ReservoirChain } from '@reservoir0x/reservoir-sdk'
+import { CSS } from '@stitches/react'
+import { ExpirationOption } from '../../types/ExpirationOption'
+import { Currency } from '../../types/Currency'
+
+type ListCheckoutProps = {
+  collection?: NonNullable<ReturnType<typeof useCollections>['data']>[0]
+  token?: NonNullable<ReturnType<typeof useTokens>['data']>[0]
+  price?: string
+  currency?: Currency
+  quantity?: number
+  chain?: ReservoirChain | null
+  expirationOption?: ExpirationOption
+  containerCss?: CSS
+}
+
+const ListCheckout: FC<ListCheckoutProps> = ({
+  collection,
+  token,
+  price,
+  currency,
+  quantity,
+  chain,
+  expirationOption,
+  containerCss,
+}) => {
+  const expirationDisplay =
+    expirationOption?.value === 'custom' && expirationOption.relativeTime
+      ? useTimeSince(expirationOption.relativeTime)
+      : `in ${expirationOption?.text.toLowerCase()}`
+
+  return (
+    <Flex align="center" justify="between" css={{ p: '$4', ...containerCss }}>
```

Comment:
does this need to be container props or can it just be props?

Reply:
Hm, I specified `containerCss` because it is specifically being destructured in the container's css property